src/contract-state.ts

```diff
@@ -60,6 +61,12 @@ export type Gateway = {
   weights: ObserverWeights;
 };
 
+export type GatewayDelegate = {
```

Comment:
Could you also modify Gateway to have:

```
delegtes: Record<string, GatewayDelegate>
```

Reply:
ah, good catch

Reply:
[ac7e924](https://github.com/ar-io/ar-io-sdk/pull/86/commits/ac7e924f24ba6133b2917de8416c37bf6560cdd4)